docs/source/usage/preparation.rst

```diff
@@ -34,22 +39,22 @@ To deploy this framework you will need the following established:
 
       See :doc:`deployment` section for more details.
 
-    4. **A CodeCommit or Github repository:**
+    5. **A CodeCommit or Github repository:**
 
       This is where your configuration files will be stored in addition to the buildspec and deploy scripts.
       If you parameterize all your sensitive data, there shouldn't be any risk of sensitive data disclosure.
 
       .. note:: The env: variables that must be filled out, these can essentially be whatever you want them to be provided they fall in line with the naming conventions the cloudformation and serverless scripts expect.
 
 
-    5. **GitHub Framework Repo:**
+    6. **GitHub Framework Repo:**
```

Comment:
Maybe here we should say: "GitHub Framework Public Repo" to ensure distinction from the custom Github repo above.

Reply:
Changed to say A CodeCommit or separate Github repository